app/models/chargeback_rate_detail_currency.rb

```diff
@@ -8,6 +8,8 @@ class ChargebackRateDetailCurrency < ApplicationRecord
 
   has_many :chargeback_rate_detail, :foreign_key => "chargeback_rate_detail_currency_id"
 
+  self.table_name = 'currencies' unless ActiveRecord::Base.connection.table_exists?('chargeback_rate_detail_currencies')
```

Comment:
I'm hoping this is right. Never done this before
But it does ensure that loading this file will not hit the database

```suggestion
  def self.load_schema!
    self.table_name = 'currencies' unless ActiveRecord::Base.connection.table_exists?('chargeback_rate_detail_currencies')
    super
  end
```